src/ReduxStore/FeedDetails/feedReducers.js

```diff
@@ -15,7 +18,10 @@ const initialState = {
   articleLibrary: [],
   popularTags: [],
   currentArticleTitle: "",
-  currentArticleSlug: ""
+  currentArticleSlug: "",
+  currentProfileData: {},
+  currentUsersArticles: [],
+  favoritedArticles:[]
 };
```

Comment:
Better to have `tagRelatedArticles` into the initialState.

Reply:
fixed